src/presentation/pages/Note.vue

```diff
@@ -54,6 +70,41 @@ const { note, noteTools, save, noteTitle, canEdit } = useNote({
   id: noteId,
 });
 
+const lastEdit = ref<string>('Last edit ');
```

Comment:
Maybe display something like "Last edit just now"? It looks strange now

Reply:
I can write `a few seconds ago` like in our finction

Reply:
Sounds good